src/ToDoItem.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const ToDoItem = ({ item, toggleCompleteItem }) => {
+    const { id, name, due, completed } = item;
+    const isOverDue = new Date(due) <= new Date() ? true : false;
```

Comment:
The `? true : false` part is redundant. First half of the code is already returning boolean value.